discord/commands/core.py

```diff
@@ -654,6 +711,10 @@ class SlashCommand(ApplicationCommand):
     description_localizations: Dict[:class:`str`, :class:`str`]
         The description localizations for this command. The values of this should be ``"locale": "description"``.
         See `here <https://discord.com/developers/docs/reference#locales>`_ for a list of valid locales.
+    integration_types: Set[:class:`IntegrationType`]
+        The installation contexts where this command is available. Cannot be set if this is a guild command.
+    contexts: Set[:class:`InteractionContextType`]
+        The interaction contexts where this command is available. Cannot be set if this is a guild command.
```

Comment:
Maybe this would be better wording (integration_types):
```suggestion
    integration_types: Set[:class:`IntegrationType`]
        The type of installation this command should be available to. For instance, if set to :attr:`IntegrationType.user_install`, the command will only be available to users with the application installed on their account. Cannot be set if this is a guild command.
    contexts: Set[:class:`InteractionContextType`]
        The interaction contexts where this command is available. Cannot be set if this is a guild command.
```